lib/hex_web/web/templates/package.html.eex

```diff
@@ -50,6 +50,11 @@ build_tools = @current_release && @current_release.meta["build_tools"] || []
           </ul>
         </dd>
       <% end %>
+
+      <%= if @hexdocs_url do %>
+        <dt>Hexdocs</dt>
```

Comment:
The conditional logic in the router seems to be out of place, but I am not
sure what would be the best place to put it.
On Nov 19, 2015 18:43, "Eric Meadows-Jönsson" <notifications@github.com>
wrote:

> In lib/hex_web/web/templates/package.html.eex
> <https://github.com/hexpm/hex_web/pull/186#discussion_r45374017>:
>
> > @@ -50,6 +50,11 @@ build_tools = @current_release && @current_release.meta["build_tools"] || []
> >            </ul>
> >          </dd>
> >        <% end %>
> > +
> > +      <%= if @hexdocs_url do %>
> > +        <dt>Hexdocs</dt>
>
> Hum, actually hold on, just saw the line below. I think we may need
> another place for this.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hexpm/hex_web/pull/186/files#r45374017>.
>